tuftool/fips-test/scripts/create-tuf-repo.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash 
+
+create_new_kms_key(){
+  alias="$1"
+  if ! [[ "$alias" =~ ^alias/ ]]; then
+    echo 'error: alias must start with "alias/"'
+    exit 2
+  fi
+
+  key_id="$(aws kms create-key --key-usage SIGN_VERIFY \
+    --customer-master-key-spec RSA_3072 \
+    --output json | jq -r .KeyMetadata.KeyId)"
+
+  aws kms create-alias --target-key-id "$key_id" --alias-name "$alias"
+}
+
+ALIAS_SUFFIX=$(date "+%Y-%m-%d")
+TEST_TUF_REPO_PATH="./test-tuf-repo"
+
+# Create RSA 3072 KMS keys
+create_new_kms_key "alias/bottlerocket-root-${ALIAS_SUFFIX}"
```

Comment:
I don't think we need to request the usage of KMS keys for the integ testing and instead we should just use ephemeral keys like the ones provided by Bottlerocket dvelopment builds.